spec/extra/empty.json

```diff
@@ -54,21 +58,22 @@
   {
     "description": "confusion with properties",
     "schema": {
-      "properties": { 
+      "properties": {
         "foo": {"type": "integer"}
       },
       "required": ["foo"]
     },
     "tests": [
       {
-        "description": "empty array validates against empty property set",
+        "description": "empty array validates against empty property set - AGAINST SPEC ???",
         "data": [],
-        "valid": false
+        "valid": true
```

Comment:
I am confused here, if this is against the spec shouldn't this be fixed instead of masking a valid test?

By the `???` I am assuming this needs more investigation.

Reply:
yep, this is one of the tests that was the reason for disabling the entire file. Hence I enabled it, and added the comment. Since it is test only, this doesn't alter the existing behaviour of the library